docs/src/cloud/cloud_faq.md

```diff
@@ -0,0 +1,34 @@
+This section provides a set of recommended best practices to help you get the most out of LanceDB Cloud. By following these guidelines, you can optimize your usage of LanceDB Cloud, improve performance, and ensure a smooth experience.
+
+### Should the db connection be created once and keep it open?
+Yes! It is recommended to establish a single db connection and maintain it throughout your interaction with the tables within. 
+
+LanceDB uses HTTP connections to communicate with the servers. By re-using the Connection object, you avoid the overhead of repeatedly establishing HTTP connections, significantly improving efficiency.
+
+### Should a single `open_table` call be made and maintained for subsequent table operations?
+`table = db.open_table()` should be called once and used for all subsequent table operations. If there are changes to the opened table, `table` always reflect the **latest version** of the data. 
```

Comment:
TBH because of the table cache, I'm not sure how important it is to re-use the table. It's definitely important for local connections, but for remote connections it doesn't matter all that much because the important caches are done server-side.

Reply:
still, it is unnecessary for users to call `open_table()` before each table op.